charms/jupyter-ui/metadata.yaml

```diff
@@ -13,7 +13,7 @@ resources:
     # TODO(knkski): Switch back to upstream after this is merged:
     # https://github.com/kubeflow/kubeflow/pull/5686
```

Comment:
With this being the new UI mentioned in the issue do we still need our own images?

Reply:
Good catch, removed

Reply:
You removed the comment but left the rocks image, should we also switch to the upstream image?

Reply:
The upstream image in `gcr.io/kubeflow-images-public/jupyter-web-app` is out-of-date compared to what's in git master. That image registry is also no longer being updated. The new registry (`public.ecr.aws/j1r0q0g6/notebooks/jupyter-web-app`) [doesn't allow listing tags](https://github.com/kubeflow/kubeflow/issues/5922), so there's no way to see if that has an updated version. The upstream Kubeflow manifests repo only [shows](https://github.com/kubeflow/manifests/blob/master/apps/jupyter/jupyter-web-app/upstream/base/kustomization.yaml#L26) `v1.3.0rc1` from that registry.

Given all that, I just pulled down the latest version from git, built it, and pushed it up with the git hash as the tag. I think that's the best course of action unless you can see a good way around any of the above.

Reply:
Sounds good to me!